installer/scripts/omsadmin.sh

```diff
@@ -404,7 +421,14 @@ unset_omsagent_proc_limit()
 
 find_current_omsagent_proc_limit()
```

Comment:
Since you've removed all usage of this method, you should delete it.

Reply:
This method is still here; are you still making changes, or do you want to keep it?

Reply:
Taking it out.